src/ex3/commands.js

```diff
@@ -0,0 +1,194 @@
+// imports
+import fs from 'fs';
+import { PokemonClient } from './pokemonClient.js';
+import { ItemManager } from './itemManager.js';
+import chalk from 'chalk';
+
+// PokemonClient instance
+const pokemonClient = new PokemonClient();
+const itemManager = new ItemManager();
+
+/* Add todo  function */
+const addTodo = async (todo) => {
+    let todoText = '';
+    let message = '';
+    if(itemManager.isPokemon(todo)){
+        const pokemon = await pokemonClient.getPokemonName(todo);
+        const type = await pokemonClient.getPokemonsTypes(pokemon);
+        todoText =`Catch ${pokemon}, a ${type} pokemon`;
+        message = `Catch ${chalk.bold(pokemon)}, a ${type} pokemon`;
+    } else{
+        todoText = todo;
+        message = "New todo added successfully!";
+    }
+    try{
+        fs.access('todos.txt', (err) => {
+            if (err){ 
+                // if todos.txt doesn't exist then create it
+                fs.writeFileSync('todos.txt', todoText);
+                console.log(chalk.bgGreen(message));
+            } else{
+                // else add todo to existiong file
+                try{
+                    fs.appendFileSync('todos.txt', `\n${todoText}`);
+                    console.log(chalk.bgGreen(message));
+                } catch{
+                    console.error(err);
+                }
+                let tasks = fs.readFileSync('todos.txt').toString().split('\n');
+            }
+        })
+    } catch(err){
+        console.error(err);
+    }
+}
+
+/* Get all the todos in todos.json */
+const getTodos = () => {
+    fs.access('todos.txt', (err) => {
```

Comment:
What happens when there's no file? Why do the user care of this error, he just needs to know that there are no pending tasks
![image](https://user-images.githubusercontent.com/12782209/173625813-85735a58-cf41-4c80-b0f6-df10486fb236.png)